package.json

```diff
@@ -7,8 +7,9 @@
   "bugs": "https://github.com/mozilla-neutrino/neutrino-dev/issues",
   "repository": "mozilla-neutrino/neutrino-dev",
   "engines": {
-    "node": ">=6.10.0",
-    "yarn": "^1.2.1"
+    "node": ">=6.9.0",
```

Comment:
The `node` engine needs to be more complex to handle the actual support. I think you can use `||` to delineate these. Also, since `7` is excluded, we probably don't want `>=` except in the `9` case.

```json
"node": "^6.9.0 || ^8 || >=9"
```

Reply:
++

Reply:
I'm pretty sure the `^8` can be written as `8` (the latter being what I've seen used elsewhere).

Reply:
I haven't seen that, but I think sticking with the `^` makes the semver relationship clearer.